lib/service/state.go

```diff
@@ -181,20 +182,24 @@ func (f *processState) readinessHandler() http.HandlerFunc {
 		case stateDegraded:
 			roundtrip.ReplyJSON(w, http.StatusServiceUnavailable, map[string]any{
 				"status": "teleport is in a degraded state, check logs for details",
+				"pid":    os.Getpid(),
```

Comment:
Can we use a `debug.Readiness` struct (or some other shared struct) here rather than a map?

Reply:
Seems like it, was unsure why a map was being used here